workitem.go

```diff
@@ -320,6 +321,10 @@ func ConvertJSONAPIToWorkItem(appl application.Application, source app.WorkItem2
 			target.Type = source.Relationships.BaseType.Data.ID
 		}
 	}
+
+	if title, ok := source.Attributes[workitem.SystemTitle]; !ok || title == "" {
```

Comment:
so I need to see if this is the good place to load the WIT from the database. Probably not..